packages/docusaurus/lib/commands/start.js

```diff
@@ -86,14 +86,12 @@ module.exports = async function start(siteDir, cliOptions = {}) {
     plugins: [
       // Generates an `index.html` file with the <script> injected.
       new HtmlWebpackPlugin({
-        inject: false,
-        hash: true,
         template: path.resolve(
           __dirname,
           '../client/templates/index.html.template.ejs',
         ),
         filename: 'index.html',
-        title: siteConfig.title,
+        title: siteConfig.title || 'Docusaurus App',
```

Comment:
Why do we put this default here? Can it be part of the default site config instead of being here?